src/style.css

```
@@ -0,0 +1,112 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.imac-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  font-size: 11px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 0 14px 15px;
+  border-radius: 5px;
+}
+
+.imac-card:hover {
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+}
+
+.imac-img {
+  display: block;
+  min-width: 160px;
+  height: 134px;
+  margin: 30px 0 45px;
+}
+
+.product-name {
+  font-size: 12px;
+  line-height: 15px;
+  font-weight: 500;
+  color: black;
+  text-decoration: none;
+  margin-bottom: 5px;
+}
+
+.product-id {
+  align-self: flex-start;
+  line-height: 13px;
+  margin-bottom: 15px;
+}
+
+.product-feedback,
+.product-price {
+  font-weight: 400;
+  line-height: 13px;
+  display: flex;
+  align-self: stretch;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.product-price {
+  margin-bottom: 20px;
+}
+
+.product-feedback {
+  margin-bottom: 20px;
+}
+
+.product-rating {
+  display: flex;
+}
+
+.rating-point {
+  background: url("./images/star-active.svg") center center no-repeat;
+  background-size: auto;
+  width: 15px;
+  height: 15px;
+  border-radius: 0.5px;
+}
+
+.rating-point.disabled {
+  background-image: url("./images/star.svg");
+}
+
+.rating-point + .rating-point {
```

Use pseudo classes if you don't want to include the first element
```suggestion
.rating-point:not(:first-child) {
```

---

why i can't use "+" sign?